__tests__/app.test.js

```diff
@@ -251,3 +251,77 @@ describe("POST /api/articles/:article_id/comments", () => {
       })
   })
 })
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: increments the votes and responds with updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            votes: expect.any(Number),
+            title: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            article_img_url: expect.any(String),
+          })
+        )
+        expect(body.article.votes).toBe(101)
+      })
+  })
```

Comment:
Nice, but the expect for votes doesnt need to be separate like this. Just as you have asserted the article_id being 1 we can assert votes in a very similar way.